model/jpa/src/main/java/org/keycloak/models/jpa/entities/ClientEntity.java

```diff
@@ -130,6 +130,12 @@
     @Column(name="BASE_URL")
     private String baseUrl;
 
+    @Column(name="BACKCHANNEL_LOGOUT_URL")
```

Comment:
Instead of changing DDL, could we just persist the new data as client attributes? Similar to what we do with `org.keycloak.protocol.oidc.OIDCConfigAttributes#JWKS_URL`.

It should help to avoid all the migration-related code herein ...

Reply:
Done, we also remove the database migration files. No need to do database changes any more.